ci/jenkins/pipelines/prs/skuba-test-vmware.Jenkinsfile

```diff
@@ -64,6 +62,12 @@ pipeline {
             sh(script: "${PR_MANAGER} update-pr-status ${GIT_COMMIT} ${PR_CONTEXT} 'pending'", label: "Sending pending status")
         } }
 
+        stage('Check for changes') { steps {
+            script {
+                env.shouldRun = !sh(script: "${PR_MANAGER} filter-pr --filename ${FILTER_SUBDIRECTORY}", returnStdout: true, label: "Filtering PR") ==~ /does not contain changes/
+            }
+        }}
+
         stage('Git Clone') { steps {
             deleteDir()
             checkout([$class: 'GitSCM',
```

Comment:
Just wondering if it makes sense to checkout the repo if we later decide not to execute any test.